contracts/Ethernauts.sol

```diff
@@ -194,6 +195,14 @@ contract Ethernauts is ERC721Enumerable, Ownable {
         beneficiary.sendValue(address(this).balance);
     }
 
+    function recoverStuckTokens(
+        address token,
+        address to,
+        uint value
+    ) external onlyOwner {
+        IERC20(token).transfer(to, value);
```

Comment:
Would be nice to add a balanceOf check here.

Some tokens don't have explicit errors, and often fail with obscure overflows. In such cases, doing the check ourselves would make these more explicit.

Reply:
Added an initial balanceOf check to make sure the owner is trying to send a realistic amount of tokens.

Initially I though about checking that the amount of recovered tokens match the amount that we moved, but in the case of deflationary tokens, this validation would fail.